04.ls/wc.rb

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  options, paths = receive_command_argument
+  paths.empty? ? output(count_stdin, options) : output(calculate_file_text(paths), options)
+end
+
+def receive_command_argument
+  opt_params = { l: false, w: false, c: false }
+  opt = OptionParser.new
+
+  opt.on('-l') { |v| opt_params[:l] = v }
+  opt.on('-w') { |v| opt_params[:w] = v }
+  opt.on('-c') { |v| opt_params[:c] = v }
+  argv = opt.parse(ARGV)
+  [opt_params, argv]
+end
+
+def count_stdin
+  inputs = $stdin.readlines.join
+  count = {
+    line: count_lines(inputs),
+    word: count_words(inputs),
+    byte: inputs.size
+  }
+  [count]
+end
+
+def calculate_file_text(paths)
+  paths.map do |file_name|
+    property = { line: 0, word: 0, byte: 0, name: '', directory: false }

```

Comment:
```suggestion
    property = { line: 0, word: 0, byte: 0 }
```

42、43行目で必ずセットされる値は省略しても大丈夫です

Reply:
承知しました。